src/github-repo-event-notifier.coffee

```diff
@@ -32,12 +32,16 @@
 #   patcon
 #   parkr
 
+_ = require('lodash')
 inspect = (require('util')).inspect
 url = require('url')
 querystring = require('querystring')
-eventActions = require('./event-actions/all')
 eventTypesRaw = process.env['HUBOT_GITHUB_EVENT_NOTIFIER_TYPES']
 eventTypes = []
+customActions = {}
+try
+  customActions = require.main.require('../../../scripts/github/custom-actions')
```

Comment:
@benoror That would make sense. Would it perhaps be better to go around all the automated inclusion (i.e. including this project name in `external-scripts.json`)? The user would add it to their `package.json`, then write a custom file in `scripts/` which includes this project and hooks it up properly for Hubot to include. That way you could add your own handlers without it being really convoluted and "magical."